monai/transforms/utils.py

```diff
@@ -1259,5 +1263,31 @@ def print_table_column(name, torch, numpy, color=Colors.none):
     print_color(f"Number of uncategorised: {n_uncategorized}", Colors.red)
 
 
+def convert_pad_mode(dst: NdarrayTensor, mode: Union[NumpyPadMode, PytorchPadMode, str]):
+    """
+    Utility to convert padding mode between numpy array and PyTorch Tensor.
+
+    Args:
+        dst: target data to convert padding mode for, should be numpy array or PyTorch Tensor.
+        mode: current padding mode.
+
+    """
+    mode = mode.value if isinstance(mode, (NumpyPadMode, PytorchPadMode)) else mode
+    if isinstance(dst, torch.Tensor):
```

Comment:
this style could be improved according to https://deepsource.io/gh/Project-MONAI/MONAI/run/d1d80c47-af73-4ac8-86c2-d5e200f7a971/python/PYL-R1705

Reply:
Sure, improved in the latest commit.
Thanks.